### PR TITLE
Cloud Video Intelligence Operators assets & system tests migration (A…

### PR DESCRIPTION
**Pull request by bkossakowska**
…IP-47)

Cloud Video Intelligence Operators assets & system tests migration (AIP-47)

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/system/providers/google/cloud/video_intelligence/example_video_intelligence.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    CloudVideoIntelligenceDetectVideoShotsOperator,
+)
+from airflow.providers.google.cloud.transfers.gcs_to_gcs import GCSToGCSOperator
+from airflow.utils.trigger_rule import TriggerRule
+
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+
+DAG_ID = "example_gcp_video_intelligence"
+
+BUCKET_NAME_SRC = "cloud-samples-data"
```
Comment (bhirsz) [inline tests/system/providers/google/cloud/video_intelligence/example_video_intelligence.py:48]: Can you create it using DAG_ID and ENV_ID variables like ``BUCKET_NAME_DST``?

Comment (bhirsz) [inline tests/system/providers/google/cloud/video_intelligence/example_video_intelligence.py:48]: Is it public repository with the sample data? Can you add comment explaining it above - for the future readers ;)

Comment (bkossakowska) [reply tests/system/providers/google/cloud/video_intelligence/example_video_intelligence.py:49]: yes, it's public bucket

Comment (bkossakowska) [reply tests/system/providers/google/cloud/video_intelligence/example_video_intelligence.py:49]: done

**Review by bhirsz — APPROVED** [review]

**Pull request closed without merging**